wordpress-mobile/WordPress-Android
# WP Stories: Add onboarding screen

Closes https://github.com/Automattic/stories-android/issues/271.

Adds a Stories onboarding screen, shown when first attempting to create a story from a FAB (and every time until the 'Create Story Post' button is tapped).

![wpandroid-stories-intro](https://user-images.githubusercontent.com/9613966/96074649-27eb5a80-0ee4-11eb-8f20-c86ce41eeb4c.png)

(Based on, and reusing styling from, the Feature Announcement feature.)

### To test:
1. From a clean install, verify that both entry points to the Story creator (my site and post list) now show the onboarding screen
2. Confirm that tapping the back arrow dismissed the screen, and trying to create a story again shows the screen again
3. Check that tapping the two story previews opens and plays each story in the browser
4. Verify that tapping the 'Create Story Post' button launches the media picker as before
5. Check that after step 4, the onboarding screen doesn't appear the next time you try to create a story
6. Check that everything works as expected with/without file/camera permissions
7. Test the screen in dark mode
8. Check on analytics (see below)

#### Analytics check:
Check that you see 'shown'/'dismissed' analytics events for the Onboarding screen when opening and dismissing it, and that the 'Create Story Post' button records its own event when tapped.

Also confirm that with or without the Onboarding screen, the `MEDIA_PICKER_OPEN_FOR_STORIES` event is recorded whenever the media picker ends up being shown, either from the My site view or the post list. (It was missing before when creating a story from the post list.)

PR submission checklist:
